services/compute/compute.go

```diff
@@ -779,6 +779,8 @@ type GalleryImageProperties struct {
 	SourceType common.ImageSource `json:"sourceType,omitempty"`
 	// CloudInitDataSource - READ-ONLY; The cloud init data source to be used with the image. [NoCloud, Azure]. Default Value – NoCloud. For marketplace images it will be Azure.
 	CloudInitDataSource common.CloudInitDataSource `json:"cloudInitDataSource,omitempty"`
+	// SizeInBytes - This property indicates the size of the gallery image in Bytes. (Int64 format)
+	SizeInBytes *int64 `json:"sizeInBytes,omitempty"`
```

Comment:
do we have this property in Azure sdk?

Reply:
It will be equivalent to this: https://github.com/Azure/azure-sdk-for-go/blob/bea25d2e2710632d5814b1a01a0b19ff10561ac6/services/compute/mgmt/2021-11-01/compute/models.go#L9061

Kept it as bytes due to the need of converting it to megabytes at the operator level.

Reply:
But GalleryImageProperties doesnt have this and we are trying to pull in the GalleryImageVersion property into this, and basically are changing the definition.
For now, can you put this in status, until we have a requirement to bring in GalleryImageVersion

Reply:
In that case, I'll move the sizeInBytes under cloudagent galleryimage to downloadstatus and modify moc/pkg/status to set the sizeInBytes as well when setting downloadStatus: https://github.com/microsoft/moc/pull/119